cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -15,44 +15,33 @@ This Cheat Sheet provides guidance on the various areas that need to be consider
 
 ### Hashing vs Encryption
```

Comment:
I tried to make this section more brief, while also giving some concrete examples about when to use hashing vs. encryption.